src/pages/product-details/product-submit/product-submit.js

```diff
@@ -93,6 +103,7 @@ const ProductSubmit = ({ setSizeIsNotSelectedError, sizes }) => {
 
   return (
     <div className={styles.submit}>
+      <Toast autoClose={3000} />
```

Comment:
Why do you use this component, if you have `ToastContainer `?

Reply:
it is deleted